account/accounts_test.go

```diff
@@ -254,14 +254,21 @@ func (s *ManagerTestSuite) TestSelectAccount() {
 			s.gethServiceProvider.EXPECT().AccountKeyStore().Return(testCase.accountKeyStoreReturn...).AnyTimes()
 			err := s.accManager.SelectAccount(testCase.address, testCase.password)
 			s.Equal(testCase.expectedError, err)
+
+			selectedWalletAccount, err := s.accManager.SelectedWalletAccount()
```

Comment:
I rather though about something like this:
```go
func (s *ManagerTestSuite) TestSelectedWalletAndChatAccountsAreEqual() {
  selectedWalletAccount, err := s.accManager.SelectedWalletAccount()
  s.NoError(err)
  selectedChatAccount, err := s.accManager.SelectedChatAccount()
  s.NoError(err)
  s.Equal(selectedWalletAccount.AccountKey, selectedChatAccount.AccountKey)
}
```

but we can also keep `func (s *WhisperTestSuite) TestSelectedChatKeyIsUsedInWhisper()`.